conference/talk_voting.py

```diff
@@ -140,6 +141,13 @@ def is_user_allowed_to_vote(user):
 def vote_on_a_talk(request, talk_uuid):
     talk = get_object_or_404(Talk, uuid=talk_uuid)
 
+    current_conference = Conference.objects.current()
+    if not current_conference.voting():
+        return HttpResponseForbidden('Voting closed.')
```

Comment:
Maybe say here "Voting not opened." in case someone hits the endpoint when voting is not opened yet.

Reply:
Well - this is more like an api endpoint rather than a full fledged view - the only way the user could see this error is if they did access the main talk voting page, and tried submitting a vote, but the talk voting was closed. Which in theory shouldn't be possible - because they shouldn't be able to reach the main talk voting page if the talk voting isn't open, but coding defensively and all that.

Reply:
got it